src/gba/core.c

```diff
@@ -272,6 +272,9 @@ static void _GBACoreLoadConfig(struct mCore* core, const struct mCoreConfig* con
 	}
 	gba->video.frameskip = core->opts.frameskip;
 
+	gba->hwExtensions.userEnabled = core->opts.hwExtensions;
+	memcpy(gba->hwExtensions.userEnabledFlags, core->opts.hwExtensionsFlags, sizeof(gba->hwExtensions.userEnabledFlags));
```

Comment:
This is where the config should be loaded, and probably better to prefix the names with `gba.` since they're GBA-specific.

Reply:
Fixed.